ivy/container/general.py

```diff
@@ -2134,3 +2135,199 @@ def clip_matrix_norm(
             map_sequences,
             out=out,
         )
+
+    @staticmethod
+    def static_stable_pow(
+        base: ivy.Container,
+        exponent: Union[Number, ivy.Array, ivy.NativeArray, ivy.Container],
+        min_base: Union[Number, ivy.Array, ivy.NativeArray, ivy.Container] = None,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.stable_pow. This method simply
+        wraps the function, and so the docstring for ivy.stable_pow also applies
+        to this method with minimal changes.
+
+        Parameters
+        ----------
+        base
+            Container of the bases of the exponentiation.
+        exponent
+            Container of the exponents of the exponentiation.
+        min_base
+            Container of the minimum base to use,
+            use global ivy._MIN_DENOMINATOR by default.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+
+        Returns
+        -------
+        ret
+            A container of elements containing the new items following the numerically
+            stable exponentiation.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.asarray([10., 15.]), b=ivy.asarray([20., 25.]))
+        >>> y = ivy.Container.static_stable_pow(x, 3)
+        >>> print(y)
+        {
+            a: ivy.array([1000., 3380.]),
+            b: ivy.array([8000., 15600.])
+        }
+
+        >>> x = ivy.Container(a=3, b=10)
+        >>> y = ivy.asarray([4, 5])
+        >>> z = ivy.Container.static_stable_pow(x, y)
+        >>> print(z)
+        {
+            a: array([81.00108001, 243.00405003]),
+            b: array([10000.04000006, 100000.500001])
+        }
+
+        >>> x = ivy.Container(a=3, b=10)
```

Comment:
This example should be removed as we should not having native arrays in the docstring examples.